tests/saw/Makefile

```diff
@@ -44,6 +46,7 @@ clean-failure-logs :
 
 
 tmp/%.log : %.saw bitcode/all_llvm.bc tmp
+	@echo "Running formal verification with ${Z3_VERSION} and ${SAW_VERSION}"
```

Comment:
Oops that was dead code. I couldn't find a way to print the saw version. We can try and add that later.